screens/MainScreen.tsx

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import {
   View,
+  ScrollView,
```

Comment:
I would vote for the photo browser should be smaller instead of making the entire activity scrollable. Or it is not possible on a tiny mobile?

Reply:
Maybe we can narrow the spacing between children.

Reply:
That is actually what will happen on normal mobiles that we get now. But I tried it on a phone which was much smaller, even though a little old. And it becomes too cramped in that case if try to keep it non scrollable. However, this change will not show up in most of the devices. I could change this if you want?

Reply:
> However, this change will not show up in most of the devices.

What do you mean by that? For instance on my iPhone 7 I can scroll the main activity but it does not make sense (everything everything fits on the screen).

Reply:
We are allowed to resize the browser to any size, there should be enough space for five _short_ components.

Reply:
> > However, this change will not show up in most of the devices.
> 
> What do you mean by that? For instance on my iPhone 7 I can scroll the main activity but it does not make sense (everything fits on the screen).

Oh is that so. Didn't happen in the 8 plus. I'll see this too.

Reply:
> We are allowed to resize the browser to any size, there should be enough space for five _short_ components.

I'm sorry could you explain?

Reply:
We have the following controls (I refer to them with their label):

1. CHOOSE A PHOTO
2. PERISHABLE GOODS
3. EXPIRATION DATE
4. SET LOCATION
5. ADD

Each of these is actually one line even the image can be like that. So we can also do a responsive layout for very small resolution.

Reply:
Ohhh okay I see. And when the image is uploaded?

Reply:
> And when the image is uploaded?

How do you mean? If the user has chosen a photo, the application should display the image snippet while keeping the aspect ratio.

Reply:
Right I've fixed this now. Works great!

Reply:
If you think you are done with every changes from the previous review then request another one, please!

Reply:
It continued here: https://github.com/ivgraai/expiry-util/pull/107#issuecomment-675425978